docs/reference/admin/layouts/slots.mdx

```diff
@@ -57,22 +65,60 @@ Using the `activeSlots` from the context you can check if there is any content t
 	{_components_SlotsSource}
 </CodeBlock>
 
-## 3. Placing slots in the layout
+## 3. Rendering page content to a slot source
 
-```tsx title="admin/components/Layout.tsx" {3,9}
-import { slotTargets, SlotTargets } from '../components/Slots'
+By wrapping the content in the slot source component, instead of rendering it as an immediate
+descendants of the page, you will create a *portal* that will render the content to the
+target component.
 
-export const Layout = ({ children: pages }: React.PropsWithChildren) => {
-  const { activeSlots } = useLayoutSlotRegistryContext()
+```tsx title="admin/pages/example.tsx" {4}
+import { SlotSources } from '../components/Slots
```

Comment:
why have you renamed Slot to SlotSource? imo it is a bit confusing and you might need to step for a few hundeds ms a think whether you need a source or a target. (and also, it is longer)

Reply:
Ambiguity: The suffix could be removed from the SlotTarget as well.